app/renderer/components/navigation/navigationBar.js

```diff
@@ -144,7 +139,8 @@ class NavigationBar extends React.Component {
     props.showHomeButton = !props.titleMode && getSetting(settings.SHOW_HOME_BUTTON)
 
     // used in other functions
-    props.navbar = navbar // TODO(nejc) remove, primitives only
+    props.isFocused = navbar.getIn(['urlbar', 'focused'])
```

Comment:
We only use it in if statement, so we don't need it, but I added it anyway :smiley: